packages/ui/src/services/breakpoints/index.ts

```diff
@@ -0,0 +1,17 @@
+import { BreakpointsConfig, ThresholdsConfig } from './types'
+
+const defaultThresholds: ThresholdsConfig = {
+  xs: 0,
+  sm: 600,
```

Comment:
```suggestion
  sm: 640,
```

640x960 is wider spread across screen size (iPhone SE for example) than 600x800 (not the same as 800x600)